worlds/dark_souls_3/__init__.py

```diff
@@ -163,6 +163,8 @@ def create_region(self, region_name, location_table) -> Region:
         if location_table:
             for name, address in location_table.items():
                 location = DarkSouls3Location(self.player, name, self.location_name_to_id[name], new_region)
+                if region_name == "Menu":
+                    location.item_rule = staticmethod(lambda item: not item.advancement)
```

Comment:
why the staticmethod wrapper?

Reply:
I refered to the value of item_rule from Location 
`item_rule = staticmethod(lambda item: True)`

I did not manage to assign the value otherwise so I probably won't be able to find another syntax in Python. 
Can you suggest something to help me ?

Reply:
if you want to exclude both useful AND advancement, you can make the location an excluded location, otherwise you probably want to use
```python
from worlds.generic/Rules import add_item_rule
add_item_rule(location, lambda item: not item.advancement)
```

Reply:
Thanks for the suggestion, including useful items in those locations are fine